src/student.js

```diff
@@ -0,0 +1,28 @@
+class Student {
+  constructor(studentId, firstName, lastName, githubUsername, email) {
+    this.studentId = 1
```

Comment:
I did `this.studentID = id` here instead, made it easier for me to assign new ID's to new students that I created